packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -58,6 +64,7 @@ export const NotifiSubscriptionContextProvider: React.FC<
   PropsWithChildren<NotifiParams>
 > = ({ children, ...params }) => {
   const [email, setEmail] = useState<string>('');
+  const [eventTypes, setEventTypes] = useState<EventTypeConfig>([]);
```

Comment:
oh hm let's not leak this all the way down

possible to call setAlertConfiguration instead from the FetchedCard? using eventTypes mapped to alert configurations